earth_enterprise/src/SConstruct

```diff
@@ -397,8 +395,6 @@ def is_version_ge(version_components, comparand_components):
       version_component = int(version_components[i])
     if int(comparand_components[i]) > version_component:
       return False
-    if int(comparand_components[i]) < version_component:
```

Comment:
Q: Why was this removed? The comparison won't work correctly, if we don't make this comparison.

Reply:
why it would not work? I tested and it works. Basically, the line above was redundant, everything else than the check that is retained is and was suppose to return TRUE and its doing so.

Reply:
It will work for the specific case of comparing 4.4.7 with 4.8, but it won't work for comparing two versions in general.

The line is not redundant.

This is comparing two numbers digit by digit starting from the most significant digit.  At each step, you either know which number is bigger, or the two digits must be equal.  The following digits have no way of telling you which number is bigger, unless all the more significant ones are equal.

You can try this for yourself:

```Python
# p.is_version_ge() is the function with the less-than check removed.
# q.is_version_ge() is the function with the less-than check.

>>> p.is_version_ge([4, 4, 7], [4, 8])
False
>>> q.is_version_ge([4, 4, 7], [4, 8])
False
# Comparing 4.4.7 and 4.8 works with the buggy version.

>>> p.is_version_ge([5, 2, 1], [4, 3])
False
>>> q.is_version_ge([5, 2, 1], [4, 3])
True
# Comparing 5.2.1 and 4.3 doesn't.
```